zebra/zapi_msg.c

```diff
@@ -2605,8 +2605,10 @@ static void zread_sr_policy_set(ZAPI_HANDLER_ARGS)
 		return;
 
 	policy = zebra_sr_policy_find(zp.color, &zp.endpoint);
-	if (!policy)
+	if (!policy) {
 		policy = zebra_sr_policy_add(zp.color, &zp.endpoint, zp.name);
+		policy->sock = client->sock;
```

Comment:
nit: Maybe move this assignment into `zebra_sr_policy_add()`?

Reply:
I am not a fan of this. the API should respect the keys for sr policy as per the RFC.
and the sock is not part of the RFC.
I prefer leaving it here.